src/pytest_workflow/workflow.py

```diff
@@ -69,6 +69,7 @@ def __init__(self,
         self._started = False
         self.errors: List[Exception] = []
         self.start_lock = threading.Lock()
+        self.desired_exit_code = 0
```

Comment:
This value should be set trough the `__init__` parameters as it should be user configurable.